### DISCUSSION TITLE
Eurobeat mod

### DISCUSSION THREAD

**Discussion by Alexa-3004** [ppy/osu#18971] [Ideas]
Similar to the nc mod, which is dt, but plays a more nightcore like version of the song, the eurobeat mod is like hr, but plays a more eurobeat like remix of the song instead

**Comment by mk56-spn** [comment]
this is.... not only incredibly niche but also just impossible to implement, eurobeat remixes are manually made things, not nightcore, which just adds some drums and keeps the songs natural pitch shift, tldr unrealistic and probably not wanted
